.github/workflows/c-cpp.yml

```diff
@@ -47,6 +47,18 @@ jobs:
           make
           sudo make install
           cd ../..
+          
+      - name: Build and Install mongoDB
```

Comment:
This is basically the whole added value from this commit, please rebase onto newest main branch